src/Command/AddUserCommand.php

```diff
@@ -44,6 +44,10 @@
  */
 class AddUserCommand extends Command
 {
+    // to make your command lazily loaded, either define its $defaultName static property,
```

Comment:
`either` does not make sense if you don't have a `or` later

Reply:
Indeed, thanks!